tox.ini

```diff
@@ -65,6 +65,7 @@ commands =
     --profile-path={toxinidir} \
     --profile=prospector \
     --die-on-tool-error {posargs}
+    flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
I believe that prospector runs flake8 behind the scenes, we just need to remove some ignored paths.

Reply:
Those paths are in some other repo so updating them is not straightforward.

Reply:
yeah, you can omit this change here for now till we update the other one (there is going to be a lot of styling and linting issues to fix)

Reply:
These are not styling issues.  This focused list of four sets of tests are about program correctness.

https://flake8.pycqa.org/en/latest/user/error-codes.html

On the flake8 test selection, this PR does  _not_ focus on "_style violations_" (the majority of flake8 error codes that [__psf/black__](https://github.com/psf/black) can autocorrect).  Instead, these tests focus on runtime safety and correctness:
* __E9__ tests are about Python syntax errors usually raised because flake8 can not build an Abstract Syntax Tree (AST).  Often these issues are a sign of unused code or code that has not been ported to Python 3.  These would be compile-time errors in a compiled language but in a dynamic language like Python, they result in the script halting/crashing on the user.
* __F63__ tests are usually about the confusion between identity and equality in Python.  Use ==/!= to compare str, bytes, and int literals is the classic case.  These are areas where __a == b__ is True but __a is b__ is False (or vice versa).  Python >= 3.8 will raise SyntaxWarnings on these instances.
* __F7__ tests logic errors and syntax errors in type hints
* __F82__ tests are almost always _undefined names_ which are usually a sign of a typo, missing imports, or code that has not been ported to Python 3.  These also would be compile-time errors in a compiled language but in Python, a __NameError__ is raised which will halt/crash the script on the user.

Reply:
yeah, what I meant is that if we update common to not exclude these files (settings/) we will need to fix styling issues reported by the prospector. Since flake8 is already run with prospector we don't need to run it again, we just need to remove the ignored paths.

Reply:
But removing the ignored paths requires modifying multiple directories because state is spread out instead of consolidated in the repo and the file that is running the tests.

Reply:
yes, that's why I'm suggesting just fixing the files for now, since the other change will require more work and coordination.